rfcs/ALL/Access Control Rules.md

```diff
@@ -0,0 +1,134 @@
+# Access Control Rules 
+
+Starting from 0.11.0 release, zenoh provides the option of controlling access to key-expressions (eg: `test/demo` ) based on network interface (eg: `lo`). The access control is managed by filtering messages (denoted as **actions** in the acl config): `put`, `get`, `declare_subscriber`, `declare_queryable`. This filteration can be applied on both sides of the interceptor: the incoming messages (`ingress`) and outgoing messages (`egress`).
+
+## Access Control Config
+
+The typical ACL configuration in the config.json5 file looks something like this:
+
+```json5
+ acl: {
+  ///[true/false] acl will be activated only if this is set to true
+  "enabled": false,
+  ///[deny/allow] default permission is deny (even if this is left empty or not specified)
+  "default_permission": "deny",
+  ///rule set for permissions allowing or denying access to key-expressions
```

Comment:
I would say "allowing or denying actions on key-expressions" instead of "allowing or denying access to key-expressions"